src/bun.js/bindings/uv-posix-polyfills.cpp

```diff
@@ -12,4 +12,22 @@ uv_pid_t uv_os_getppid()
     return getppid();
 }
 
+#if OS(LINUX)
+#include <time.h>
+
+uint64_t uv_hrtime()
+{
+    timespec ts;
+    clock_gettime(CLOCK_MONOTONIC, &ts);
+    return ts.tv_sec * 1000000000ULL + ts.tv_nsec;
+}
+
+#elif OS(DARWIN)
+#include <mach/mach_time.h>
+uint64_t uv_hrtime()
+{
+    return mach_absolute_time();
```

Comment:
libuv uses `mach_continuous_time` and uses `mach_timebase_info` with the result.
https://github.com/libuv/libuv/blob/v1.x/src/unix/darwin.c#L62

Reply:
`continuous` [increments](https://developer.apple.com/documentation/kernel/1646199-mach_continuous_time) while the system is asleep and `absolute` [doesn't](https://developer.apple.com/documentation/kernel/1462446-mach_absolute_time)